### ISSUE TITLE
Improve the token selector

### ISSUE THREAD

**Issue by Giorgionocera** [bitsongofficial/sinfonia-ui#385]
At the moment, we are using a select (dropdown) to let the user select a token in the swap component. Due to the fact that we have a big number of tokens (which number should increase soon), we should propose a new component for the token selection.

Some suggestions:
- We can change the dropdown with a dialog
- We can add a search bar inside the dialog
- We can add other info for each asset (Available asset, price, etc.)
- We can sort by available balances

Some references:
- Uniswap
- Astroport

We need the design for such a component (also the swap one).
![swap-component](https://user-images.githubusercontent.com/13519292/178730056-86529087-5f3f-4b29-bddb-436ad6e430a4.jpg)

**Comment by giannnni** [comment]
Due to a redesign of the entire Swap Component, I could suggest a better solution that could include some extra help for users like a search field and a small "Recent Swapped Token" carousel. 

You could see the new select component just below:
![image](https://user-images.githubusercontent.com/92384659/178735624-702ac005-4a8c-4fa9-84e9-9b96ceea21cd.png)

**Comment by Giorgionocera** [comment]
I think it is a nice solution. Thank you!
Do you have also the mobile and tablet versions?

Anyway, let's see what @angelorc thinks about it.

**Comment by giannnni** [comment]
Yes, you could see here a minimized version for mobile only (opened as a bottom sheet): 

· Base version: 
![image](https://user-images.githubusercontent.com/92384659/178995235-706c97a8-9c7c-476a-9f4a-565cf0059e42.png)
· Slippage menu open: 
![image](https://user-images.githubusercontent.com/92384659/178995342-ca4f9545-e55f-4a86-88d1-9456cb25ad9c.png)
· Select Token open: 
![image](https://user-images.githubusercontent.com/92384659/178995439-5e8769af-3712-4231-8239-982adadaed80.png)

**Comment by Giorgionocera** [comment]
It is very nice. However I think this blur could be an issue at the moment, but let's wait for feedback from @DavideSegullo.

**Comment by DavideSegullo** [comment]
@Giorgionocera It's better if we doesn't use it if not required

**Comment by Giorgionocera** [comment]
@giannnni, can we remove it? Or is it required?

**Comment by giannnni** [comment]
Ok, let's move to solid color #311D3E

**Comment by Giorgionocera** [comment]
As from the meeting with @angelorc, we need:
- [ ] reduce margins in tokens selector;
- [ ] move the FanToken name under the symbol.
So please @giannnni , let's perform these edits! 💪

**Comment by giannnni** [comment]
I fixed the table reducing margins and moving fantoken name under symbol. 
Tell me if it's enough or you need less margins (I suggest not).

Here my proposal: ![image](https://user-images.githubusercontent.com/92384659/180458622-7ef762fb-d772-448e-853b-2c7201a3e6aa.png)

**Comment by Giorgionocera** [comment]
Yeah, I think it could be nice! Thank you @giannnni !
@DavideSegullo we can move it in development!

**Comment by Giorgionocera** [comment]
This also includes issue #335.

**Comment by Giorgionocera** [comment]
Thank you!

**Comment by Giorgionocera** [comment]
I think that in the desktop version the information on the amount of token available should be visible at every moment of the swap action. We need to bring it back.
